tests/test_core.py

```diff
@@ -24,48 +26,60 @@ def setUp(self):
         self.gps_inf = -1000.0
 
     def tearDown(self):
+        time.sleep(20)
         os.close(self.db_fd)
         os.unlink(self.db_name)
 
     def test_0a_disable(self):
+        time.sleep(20)
```

Comment:
When you have to do the same at the beginning of all your tests... who do you call?